doc/GettingStarted.md

```diff
@@ -122,6 +125,64 @@ In addition, Vitess requires the software and libraries listed below.
     ``` sh
     $ sudo apt-get install openjdk-7-jre
     ```
+    
+#### OS X
+
+1.  [Install Homebrew](http://brew.sh/).
+
+2.  On OS X, MySQL 5.6 has to be used, MariaDB doesn't work for some reason yet. It should be installed from Homebrew
+    (install steps are below).
+   
+3.  ZooKeeper is used as lock service.
+
+4.  Run the following commands:
+
+    ``` sh
+    brew install go automake libtool memcached python mercurial git bison curl wget homebrew/versions/mysql56
+    pip install --upgrade pip setuptools
+    pip install virtualenv
+    pip install MySQL-python
+    ```
+    
+5.  Install Java runtime from this URL: https://support.apple.com/kb/dl1572?locale=en_US
```

Comment:
This seems to be an old Java 6 runtime. Is there any reason it won't work with a newer one like from here?

http://www.oracle.com/technetwork/java/javase/downloads/index.html